mne/time_frequency/tests/test_tfr.py

```diff
@@ -264,7 +276,12 @@ def test_tfr_multitaper():
                                             time_bandwidth=4.0, picks=picks)
     power_evoked = tfr_multitaper(epochs.average(), freqs=freqs,
                                   n_cycles=freqs / 2., time_bandwidth=4.0,
-                                  return_itc=False)
+                                  return_itc=False, average=False).average()
```

Comment:
> why have average param at all if it's so easy just to do .average() at the end?

If average == True the memory load is `n_chans,  n_freqs, n_times`, whereas if only average at the end the memory load goes through `n_epochs, n_chans,  n_freqs, n_times`. Also, it is not possible to get some metrics by averaging a posteriori, e.g. inter-trial coherence

@choldgraf IIUC these tests are just to check that the average over single trial estimates are numerically identical to the average estimate. This is important to check because they are computed differently in `_tfr_aux`

@agramfort can you confirm and add descriptive comments accordingly?